app/build.gradle

```diff
@@ -176,6 +176,7 @@ android {
         }
     }
     compileOptions {
+        coreLibraryDesugaringEnabled true
```

Comment:
In my experience, desugaring doesn't successfully make the app work in old phones. Please implement without needing the new Date library

Reply:
OK. I did it another way.